recipes/sqlitecpp/config.yml

```diff
@@ -3,4 +3,4 @@ versions:
     folder: all
   "2.5.0":
     folder: all
-  
\ No newline at end of file
+  
```

Comment:
There's a few false positives. However fixing all of there' seems much better

Reply:
This file has final whitespace instead of a final newline.
I fixed my script to handle this.